employees/tests/test_unit_report_viewset.py

```diff
@@ -430,130 +433,75 @@ def test_serializer_project_field_queryset_should_contain_only_projects_user_is_
 
 class ReportCustomDetailTests(TestCase):
     def setUp(self):
-        self.user = CustomUser(
-            email="testuser@codepoets.it", password="newuserpasswd", first_name="John", last_name="Doe", country="PL"
-        )
-        self.user.full_clean()
-        self.user.save()
-
-        self.project = Project(name="Test Project", start_date=datetime.datetime.now())
-        self.project.full_clean()
-        self.project.save()
+        super().setUp()
```

Comment:
After refactoring to django view please move tests related to this view to `test_views.py`.

Reply:
Done